dirty_cat/datasets/tests/test_fetching.py

```diff
@@ -65,12 +66,14 @@ def test_fetch_openml_dataset():
                                                  data_directory=test_data_dir)
 
         except URLError:
-            # No internet connection, or the website is down.
-            # We abort the test.
-            #
-            # One could try to manually recreate the tree structure
-            # created by ``fetch_openml()``,  and the
-            # ``.gz`` files within in order to finish the test.
+            warnings.warn(
+                "No internet connection or the website is down, test aborted."
+            )
+            pytest.skip(
+                "One could try to manually recreate the tree structure "
+                "created by ``fetch_openml()``,  and the "
+                "``.gz`` files within in order to finish the test."
```

Comment:
I think that the content of this should go in a comment, and the message in the "skip" call should be more understandable to someone who is looking only at the logs. Something like "URLError: skipping this test as the failure might be due to an internet connection problem".

Reply:
Ah, I see ! I'll change that